### PR TITLE
Test with JRuby 9000 on travis

### LINKED ISSUE #27: Add JRuby 9000 to supported Ruby versions
At the moment the tests works just fine with JRuby 9.0.4.0.

```
...
Finished in 2.72 seconds (files took 2.21 seconds to load)
41 examples, 0 failures
```

### PR DESCRIPTION
**Pull request by roback**
close #27

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 rvm:
-- 1.9.3
-- 2.0.0
-- 2.1
-- 2.2
+  - 1.9.3
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - jruby-head
```
Comment (walro) [inline .travis.yml]: Can we keep up with this? I think we should lock to a set of versions if we can.

Comment (roback) [inline .travis.yml]: You're right, its probably best to lock to some versions. I'll add the first version we support (`jruby-9.0.0.0`) and the latest (`jruby-9.0.4.0`) instead.

**Comment by roback** [comment]
Where did travis go? The tests didn't run on the last commit. :confused:

**Comment by walro** [comment]
Perhaps related to https://github.com/travis-ci/travis-ci/issues/4720?

**Comment by roback** [comment]
Maby. I expected travis to at least try to start a build though.

**Comment by walro** [comment]
Now they showed up :)

**Comment by walro** [comment]
> Unknown ruby string (do not know how to handle): jruby-9.0.4.0.
Found remote file https://s3.amazonaws.com/jruby.org/downloads/9.0.4.0/jruby-bin-9.0.4.0.tar.gz

Seems to figure that out too.

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-- 1.9.3
-- 2.0.0
-- 2.1
-- 2.2
+  - 1.9.3
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - jruby-9.0.0.0
+  - jruby-9.0.4.0
```
Comment (walro) [inline .travis.yml]: Seems to be super-slow with 9.0.4.0. For me it's ok if we just run with 9.0.0.0.

Comment (roback) [inline .travis.yml]: Yes but it worked :) 

I'll remove 9.0.4.0, if it works on 9.0.0.0 it should work on the newer version too.

File: README.md [hunk 3]
```diff
@@ -42,7 +42,9 @@ Too learn more about the capabilities of this API you should read the [Twingly S
 ## Requirements
 
 * API key, contact sales@twingly.com to get one
-* Ruby 1.9, 2.0, 2.1, 2.2
+* Ruby
+  * Ruby 1.9, 2.0, 2.1, 2.2
+  * JRuby >= 9.0.0.0
```
Comment (walro) [inline README.md]: Change "JRuby >= 9.0.0.0" to just "JRuby 9000"?

**Comment by walro** [comment]
LGTM!

**Pull request merged**